src/StackHCI/custom/stackhci.ps1

```diff
@@ -953,12 +955,37 @@ function Install-Dependent-Module{
             Install-PackageProvider NuGet -Force | Out-Null
         }
 
-        Write-VerboseLog ("Installing Module: {0} version: {1}" -f $ModuleName,$ModuleVersion)
+        Write-VerboseLog ("Installing Module: {0}" -f $ModuleName)
 
-        Install-Module -Name $ModuleName  -RequiredVersion $ModuleVersion  -Force -AllowClobber -Repository 'PSGallery'
-        Import-Module -Name $ModuleName -RequiredVersion $ModuleVersion
+        Install-Module -Name $ModuleName -Force -AllowClobber -Repository 'PSGallery'
+        Import-Module -Name $ModuleName
         
-        Write-VerboseLog ("Successfully imported Module: {0} version: {1}" -f $ModuleName,$ModuleVersion)
+        Write-VerboseLog ("Successfully imported Module: {0}" -f $ModuleName)
+    }
+}
+
+function Check-DependentModules 
+{
+    param()
+
+    $missingDependentModules = [System.Collections.ArrayList]::new()
+
+    if (-Not (Get-InstalledModule -Name "Az.Accounts" -MinimumVersion $AzAccountsModuleMinVersion -ErrorAction SilentlyContinue))
```

Comment:
Because Az.Accounts must have been imported before this script executes, it makes more sense to check for the imported version rather than the installed version, something like:
```powershell
$accountsName = 'Az.Accounts'
$accountsModule = Get-Module -Name $accountsName
if(-not $accountsModule) {
    ......
} elseif (($accountsModule.Version -lt [System.Version]'2.10.2')) {
    ......
}
```

Reply:
Sure, updated